W-13729475 setup desktop duke (#78)
mulesoft/docs-code-builder: modules/ROOT/pages/mock-data-using-dw-library.adoc

== Import the DataWeave Library

. Open the Command Palette by pressing Ctrl+Shift+p (Windows) or Cmd+Shift+p (Mac), and provide the following command:
+
[source,command]
----
MuleSoft: Import Asset from Exchange
----
+
image::import-asset-from-exchange-db.png[]
. Select *DataWeave Library*.

> . Start typing _DataWeave_, and press Return for Anypoint Code Builder to suggest the assets that match your search.

Author: ```suggestion
. Start typing _DataWeave_, and press Return to list assets that match your search.
```
